code/modules/medical/diseases/plasmatoid.dm

```diff
@@ -7,3 +7,54 @@
 	recureprob = 15
 	associated_reagent = "liquid plasma"
 	affected_species = list("Monkey", "Human")
+
+/datum/ailment/disease/plasmatoid/stage_act(mob/living/affected_mob, datum/ailment_data/D)
+	if (..())
+		return
+	switch(D.stage)
+		if(2)
+			if(prob(1))
+				affected_mob.emote("cough")
+			if(prob(1))
+				boutput(affected_mob, "<span class='alert'>Your muscles ache.</span>")
```

Comment:
Messages feel fairly unrelated to your lungs being forcefully replaced by a foreign virus

Reply:
Made some updates, felt like it was more of a transformation but at least added some feedback for when it happens.